docs/docs/components/button.md

```diff
@@ -37,6 +37,16 @@ disabled: boolean = false;
 // By default, opacity of a disabled element is 0.5. This value can be overriden with this property
 disabledOpacity: number = undefined;
 
+// Should be focused when the component is mounted, see also View's arbitrateFocus
+// property.
+// WARNING: autoFocus=true means that this Button's focus() method will be called,
+// however calling focus() might have no effect (for example the button is disabled),
+// the application has to handle this either while setting this property or in the
+// FocusArbitrator callback.
+// NOTE: The focus() call will be performed on the next tick after the current
```

Comment:
Is it necessary to document that it "will be performed on the next tick..."? This seems like an implementation detail that is unimportant to document. If we do document it, we'll be constrained to implement it this way on all platforms in the future.